### ISSUE TITLE
Generated code doesn't match generated table model when using aliases for automatic timestamp fields

### ISSUE THREAD

**Issue by SignedAdam** [volatiletech/sqlboiler#788]
If you're having a generation problem please answer these questions before submitting your issue. Thanks!

### What version of SQLBoiler are you using (`sqlboiler --version`)?
v3.7.1

### What is your database and version (eg. Postgresql 10)
Postgresql 12.2

### Please provide a relevant database schema so we can replicate your issue (Provided you are comfortable sharing this)
```
CREATE TABLE IF NOT EXISTS "user_management"."user"
(
    "id"                uuid                        PRIMARY KEY,
    "created_by"        uuid                        NOT NULL,
    "created_at"        timestamp with time zone    NOT NULL,
    "updated_by"        uuid                        NOT NULL,
    "updated_at"        timestamp with time zone    NOT NULL,
    "deleted"           bool                        NOT NULL,
    "deleted_by"        uuid                        NOT NULL,
    "deleted_at"        timestamp with time zone    NOT NULL,
    "first_name"        varchar(100)                NOT NULL,
    "last_name"         varchar(100)                NOT NULL,
);
```

### Further information. What did you do, what did you expect?
In the sqlboiler.toml, I set the aliases for the automatic time fields as such:
```
[aliases.tables.user.columns]
    created_at = "CreateTime"
    updated_at = "UpdateTime"
    deleted_at = "DeleteTime"
```

and after code generation the table model looks correct:
```	
type Customer struct {
	...
        CreateTime   time.Time  `boil:"created_at" json:"created_at" toml:"created_at" yaml:"created_at"` 
	...
}
```
However in all the database interaction functions, the old time field names are used
```
func (o *User) Insert(ctx context.Context, exec boil.ContextExecutor, columns boil.Columns) error {
	if o == nil {
		return errors.New("models: no user provided for insertion")
	}

	var err error
	if !boil.TimestampsAreSkipped(ctx) {
		currTime := time.Now().In(boil.GetLocation())

		if o.CreatedAt.IsZero() {                          <<<
			o.CreatedAt = currTime                     <<<
		}
		if o.UpdatedAt.IsZero() {                          <<<
			o.UpdatedAt = currTime                     <<<
		}
	}
        ...

```

the code obviously doesn't compile

Is there a solution to this while keeping the aliases?

I use reflection to automatically map my input structs to the table model generated by sqlboiler and would like to stick to google standardised names for fields like created_time, updated_time, ...

**Comment by aarondl** [comment]
I suppose we've never had someone attempt to rename these fields using aliases. Clearly just a bug with the generation of the automated timestamp code (they aren't taking into account the aliases I'm guessing).

I can't see a solution where you get to keep the automated timestamp functionality along with the alias. Your best bet is to either special-case your reflection code or turn off automated timestamps.

This is a low priority issue (as its basically a personal fix for your own preference and isn't really blocking) and as such I'm probably not going to take a look at it for some time until there's a bunch of these kinds of issues that need dealing with. Would accept a PR much more quickly. Thanks for reporting the issue.

**Comment by stephenafamo** [comment]
Fixed in #834   
It is now also possible to customize the auto timestamp fields, implemented by #962
